packs/data/bestiary-effects.db/effect-alchemical-crossbow.json

```
@@ -0,0 +1,69 @@
+{
+    "_id": "ucAwSIZsGrpBLg6G",
+    "data": {
+        "badge": null,
+        "description": {
+            "value": "<p>Vilree's crossbow can deliver bolts that are alchemically infused. As an action, she can slot an alchemical bomb into the crossbow, and the next 3 attacks she makes with it deal 1d6 damage of the type dealt by the bomb. These attacks never deal splash damage.</p>"
```

```suggestion
            "value": "<p>The Alchemical Crossbow deals an additional 1d6 damage of the type dealt by the bomb used.</p>"
```
Effects should simply say what they do, without other details in them that may lead people to think that something like the 3 attack thing is automated.

Also we'll need all the other possible damage types added. While the NPC only has a few to pick from we have alchemical bombs of basically all damage types now so it's going to be a large gross list but nothing in the rules says you can't slap a peshpine into the thing so 🤷.

---

I was thinking that we could just add the REs to the actual equipment for PCs, and possibly make it so that the player has to choose the actual alchemical item in their inventory; the only reason I'm not doing that for this NPC is due to NPCs not using weapons.